python/samples/requirements.txt

```diff
@@ -1,7 +1,7 @@
 ../sqlcommenter-python[opentelemetry]
-opentelemetry-api==0.16b1
-opentelemetry-exporter-google-cloud==0.16b1
-opentelemetry-instrumentation-psycopg2==0.16b1
-opentelemetry-sdk==0.16b1
+opentelemetry-api~=1.0
+opentelemetry-exporter-gcp-trace==1.0.0
+opentelemetry-instrumentation-psycopg2==0.23b2
+opentelemetry-sdk~=1.0
```

Comment:
Since this is an example, should we pin all the dependencies and let dependabot upgrade them?

Reply:
I agree! Done!